buildkite/build_release_pipeline.yml

```diff
@@ -0,0 +1,8 @@
+steps:
+  - label: ':package: Build'
+    command:
```

Comment:
For the other projects I always added a `git clean -ffdxq .` at the top to be sure that nothing old was left over. Though of course this breaks any caching, but I would argue that knowing the script will always take the same path is worth this - especially if the task is only run once a day.

Reply:
OK, let's try.